src/dndbeyond/content-scripts/character.js

```diff
@@ -1249,6 +1258,11 @@ function rollSpell(force_display = false, force_to_hit_only = false, force_damag
         if (healing_modifiers.length > 0) {
             handleSpecialHealingSpells(spell_name, damages, damage_types, {spell_level: level, spell_source, castas});
         }
+        else {
+            to_hit = handleSpecialGeneralAttacks(damages, damage_types, properties, settings_to_change, {to_hit, spell_name, spell_level: level});
+        
+            handleSpecialSpells(spell_name, damages, damage_types, {spell_level: level, spell_source, castas});
+        }
```

Comment:
Why is `handleSpecialSpells` inside the `else` so it's only for non healing spells ? I think it should be outside the else (even if there is nothing that would apply for it in non-healing spells)
Also did the condition that made you call `handleSpecialGeneralAttacks` for non-hunters-mark not apply anymore ?